### PR TITLE
Adding more programs from src/cmd

### PR DESCRIPTION
**Pull request by elbing**
Added:
execnet
bc (doesn't work properly, help needed)
units
vt (segattach always returns (void *)-1)

Removed and added to APEX repo:
bzip2recover
compress
eqn

### CODE REVIEW

File: sys/src/cmd/bc/bc.y [hunk 1]
```diff
@@ -29,7 +29,7 @@
 	char*	pre;
 	char*	post;
 
-	long	peekc = -1;
+	int32_t	peekc = -1;
```
Comment (rminnich) [inline sys/src/cmd/bc/bc.y]: This really ought to be an int if possible. Do we really have to say int32 in this case? It's just a char.

File: sys/src/cmd/bc/bc.y [hunk 2]
```diff
@@ -77,11 +77,12 @@
 	void	pp(char*);
 	void	routput(char*);
 	void	tp(char*);
+	int yylex(void);
 	void	yyerror(char*, ...);
 	int	yyparse(void);
 
 	typedef	void*	pointer;
-	#pragma	varargck	type	"lx"	pointer
+	/* #pragma	varargck	type	"lx"	pointer */
```
Comment (rminnich) [inline sys/src/cmd/bc/bc.y]: If this will never ever be used, kill it.

File: sys/src/cmd/bc/bc.y [hunk 3]
```diff
@@ -773,7 +773,7 @@ cpeek(int c, int yes, int no)
 int
 getch(void)
 {
-	long ch;
+	int32_t ch;
```
Comment (rminnich) [inline sys/src/cmd/bc/bc.y]: getch returns an int. Why is this int32_t?

File: sys/src/cmd/units/units.y [hunk 4]
```diff
@@ -18,7 +18,7 @@ typedef	struct	Prefix	Prefix;
 struct	Node
 {
 	double	val;
-	schar	dim[Ndim];
+	uint8_t	dim[Ndim];
```
Comment (rminnich) [inline sys/src/cmd/units/units.y]: schar is signed char. Why is this unsigned?

File: sys/src/cmd/bc/bc.y (hunk 1 above)
Comment (elbing) [inline sys/src/cmd/bc/bc.y]: How could a char be a long in the original?

File: sys/src/cmd/bc/bc.y (hunk 2 above)
Comment (elbing) [inline sys/src/cmd/bc/bc.y]: Getting a knife...

File: sys/src/cmd/bc/bc.y (hunk 3 above)
Comment (elbing) [inline sys/src/cmd/bc/bc.y]: In harvey we converted long to int32 and vlong to int64. I'll put an int there.

File: sys/src/cmd/units/units.y (hunk 4 above)
Comment (elbing) [inline sys/src/cmd/units/units.y]: Because my brain saw there  "short char". I never saw "schar" until now (2017).

**Comment by elbing** [comment]
Any idea about why bc doesn't work? I doubt that types could mark a difference.

**Comment by bldybuild** [comment]
Commit cd814b7d8264f9fcf4383be064bf4d84d5389046 is not signed-off.

Please fix these issues:

>	1. Álvaro Jurado has not signed-off cd814b7.


If you'd like more information on how to sign your commits please visit [signed-off-by.me](https://signed-off-by.me)

**Pull request merged**